### ISSUE TITLE
Clean up the recipes.

### ISSUE THREAD

**Issue by jamesplease** [MarionetteLabs/marionette-cookbook#20]
Right now the cookbook is a bit of a mess, but that's to be expected as we get started.

This is just to note that one day we will need to clean this up. We should come up with rules to decide whether or not something is a good recipe. Should there be duplicate recipes? What's the requirement for a recipe to be acceptable?

Just some thoughtz

**Comment by joshbedo** [comment]
Agreed, I like the idea of "This is one way to solve the problem but not the only way" solutions. In that aspect I think it's okay to have 1 or 2 solutions but 1 will probably always be more flexible/reusable than the latter eliminating competing solutions.
